packages/uniforms-antd/src/BoolField.tsx

```diff
@@ -12,6 +13,7 @@ export type BoolFieldProps = Override<
     checkbox?: boolean;
     inputRef?: Ref<HTMLInputElement>;
     onChange(value?: boolean): void;
+    onReset?: () => void;
```

Comment:
Why is this change needed?

Reply:
Removing this line cause 
```ts
Type '(event: FormEvent<HTMLDivElement>) => void' is not assignable to type '() => void'
```
in wrapper props.
Seems like `FormItemProps` does not satisfy all input fields.

Reply:
Ah, OK. Use `Omit<HTMLProps<HTMLDivElement>, 'onReset'>` then.